eva/expression/function_expression.py

```diff
@@ -68,6 +68,13 @@ def name(self):
     def output(self):
         return self._output
 
+    @property
+    def col_alias(self):
```

Comment:
Are we merging all the column names?

Reply:
Yes. Do you have other suggestions? Not sure what is a good way to create the alias for multi-columns together.

Reply:
This is tricky. We allow FunctionExpression to produce multiple columns, so we cannot assign a single column to it. Ideally, we should check in the binder if the UDF composition is wrong. For now, you can iterate over the list and check for each column

Reply:
You mean iterate and check inside the orderby executor?

Reply:
Yes

Reply:
Yes